docs/adr/0013-interactive-templates-repo.md

```diff
@@ -0,0 +1,51 @@
+# 13. Use external repo for interactive code schema and templates
+Date: 2023-03-24
+
+## Status
+Accepted
+
+## Context
+
+We need to support dynamically generating code from user input as part of OSI.
+Functionally, this has two main parts: 1) the form the user fills in, and 2 analysis
+code templates to be rendered with the form data.
+
+These two parts are tightly coupled - changes to the form data schema needs to
+match changes in the templates.
+
+However, whilst the form itself must live in job-server repo, there are reasons
+having the templates in job-server repo is problematic:
```

Comment:
```suggestion
However, whilst the form itself must live in the job-server repo, there are reasons that make doing so problematic:
```

Reply:
This changes the meaning? Gonna fix the formatting myself

Reply:
Ah, I missed some words!  It was the dual use of `job-server repo` in the same sentence that I was trying to avoid.  How about:

> However, whilst the form itself must live in the job-server repo, there are reasons that make having the templates there problematic: